pkg/tenant/db/projects.go

```diff
@@ -102,6 +102,11 @@ func CreateTenantProject(ctx context.Context, project *Project) (err error) {
 		return err
 	}
 
+	// Store the project tenant ID as a key and project org ID as a value in the database for org verification.
+	if err = PutOrgIndex(ctx, project.TenantID, project.OrgID); err != nil {
+		return err
+	}
```

Comment:
This is `CreateTenantProject` right? Shouldn't this be:

```suggestion
	if err = PutOrgIndex(ctx, project.ID, project.OrgID); err != nil {
		return err
	}
```

? We're not creating the tenant, but rather the project, unless I'm misunderstanding something. If we're also creating the tenant, then we also need to associate the tenant with the organization and need a second `PutOrgIndex` call for that resource.